autoload/tagbar.vim

```diff
@@ -3051,6 +3051,7 @@ function! s:run_system(cmd, version) abort
         exec pyx . '__argv = {"args":vim.eval("a:cmd"), "shell":True}'
         exec pyx . '__argv["stdout"] = subprocess.PIPE'
         exec pyx . '__argv["stderr"] = subprocess.STDOUT'
+        exec pyx . '__argv["errors"] = "ignore" '
```

Comment:
```suggestion
        exec pyx . '__argv["errors"] = "ignore"'
```

Is that a stray space at the end of this string? I assume this array gets joined or passed as an array and doesn't need trailing spaces, no?